autoload/utils/common.vim

```diff
@@ -4,6 +4,7 @@
 " Executes the command
 function! utils#common#executeCommand(cmd, ...) abort
     " Close quickfix list in order to don't save custom error format
+    " is this necessary?
```

Comment:
As I remember without the closing buffer the plugin will have some issues with error parsing (I don't know how it works with job, but I faced with this issue when I have used Dispatch).